src/main/java/org/jnode/fs/FileSystemType.java

```diff
@@ -51,7 +53,7 @@
      * @throws FileSystemException if error occurs during creation of the new
      *             file system.
      */
-    public T create(Device device, boolean readOnly) throws FileSystemException;
+    public T create(Device device, boolean readOnly) throws FileSystemException, IOException;
```

Comment:
```suggestion
    public T create(Device device, boolean readOnly) throws FileSystemException;
```
This interface shouldn't change/doesn't need to change - this function signature should be usable as is.

Reply:
The suggested change was applied, thanks